app/mailers/transactions_mailer.rb

```diff
@@ -11,11 +11,13 @@ def automatic_top_up_done(transaction)
     mail to: @user.email, title: "Automatic top up account - #{@to_account.name}"
   end
 
-  def transaction_notification(account, account_share = nil)
+  def transaction_notification(account, recipients)
+    return if recipients.empty?
```

Comment:
should we do `prevent_delivery and return` as above?
https://github.com/widefix/pocketmoney/blob/885a4a6e7cf550fb15ed8e99e0431e7dd7e4f048/app/mailers/transactions_mailer.rb#L9C5-L9C21

Reply:
Actually, does the `prevent_delivery` work? I can't find such a method in ActionMailer.

Reply:
It works. Method is defined in ApplicationMailer